src/one_dragon/base/push/channel/q_msg.py

```
@@ -0,0 +1,120 @@
+        try:
+            key = config.get('KEY', '')
+            msg_type = config.get('TYPE', 'send')
+
+            ok, msg = self.validate_config(config)
+            if not ok:
+                return False, msg
+
+            # 构建请求URL
+            url = f'https://qmsg.zendee.cn/{msg_type}/{key}'
+
+            # 处理消息内容：替换 ---- 为 -，并合并标题和内容
```

Ruff (0.14.0)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)